ServerCore/Pages/EventSpecific/FAQ.cshtml

```diff
@@ -5,6 +5,27 @@
     ViewData["AdminRoute"] = "/EventSpecific/FAQ";
     ViewData["AuthorRoute"] = "/EventSpecific/FAQ";
     ViewData["PlayRoute"] = "/EventSpecific/FAQ";
+    var eventPage = Model as ServerCore.ModelBases.EventSpecificPageModel;
+    var Event = eventPage?.Event;
 }
 
-<partial name="../Resources/@(Model.Event.HomePartial ?? "Default")/FAQPartial.cshtml" />
\ No newline at end of file
+@if ((Event.EventBegin.Year < 2024) || string.IsNullOrEmpty(Event.HomePartial))
+{
+    if (!string.IsNullOrEmpty(Event.FAQContent))
```

Comment:
Why not support FAQContent whenever it is specified? I happen to know of a 2024 event that is using it today. :-) Ditto the other two pages of course.

Reply:
If we still plan on using that field we can; I just thought it was no more or less difficult to add code there than it is to a separate file you'll upload and that we didn't need both ways of doing things.  The question is:  what hierarchy do we want?  I wasn't really a fan of this field completely overwriting the partials to begin with, so we should decide right now:  do we ever only show one set of content, or can we have both the database field AND the custom html file show up at the same time?  

We should also decide how the partials will be handled.  I know you also commented on one of the partial files below, but I'll continue this conversation here in case future changes to the partial files mess with your comment, as I know github sometimes does :)  I believe that we no longer need partials at all and that we should stop using them for all future events immediately, including rubix, ph24, and pd2024.  They require re-deployment of the site for new events, which this PR is explicitly trying to stop.  

Keeping in mind that the partials themselves for old events are the ones that handle including the event database field, and that these default files are only ever shown to admins who will otherwise have access to the wiki I'm writing anyway, I propose the following:
- Remove the check for the HomePartial field.  This will mean old events will always get the partials, and new events won't at all.  
- Move the event database field into the else block so new events get both it and the html file.  If both are missing, you just get no content, which is totally fine since that default content was just a failsafe anyway.  

Note that we're still using the HomePartial field for old events to also apply the style files located in the wwwroot folder, so we'll likely never be able to get rid of that field, but we now no longer will need to set it at all for any future events since no new events will have partial files associated with them.